jenkinsfiles/android_jenkinsfile.groovy

```diff
@@ -1,13 +1,31 @@
 /**
 * Android Jenkinsfile
 */
-node('android') {
-    stage 'Checkout'
+node("android"){
+  stage("checkout"){
     checkout scm
+  }
 
-    stage 'Build'
-    sh "./gradlew clean assembleDebug"
+  stage("build"){
+    sh 'chmod +x ./gradlew'
+    sh './gradlew clean assembleRelease' // builds app/build/outputs/app-release-unsigned.apk file
+  }
 
-    stage 'Archive'
-    archiveArtifacts artifacts: 'app/build/outputs/apk/*.apk', excludes: 'app/build/outputs/apk/*-unaligned.apk'
-}
+  stage("sign"){
```

Comment:
this should be only invoked when the build is built for release. You can use "BUILD_CONFIG" build param to check if the build is for debug or release